### PR TITLE
Unused code with unclear intent

### PR DESCRIPTION
**Pull request by vporton**
Added TODO comments for a not best-practice code. I can't fix it myself, because I do not understand code intent.

### CODE REVIEW

File: server/src/load_data.py [hunk 1]
```diff
@@ -25,6 +25,7 @@
 redis_conn = redis.from_url(config.REDIS_URL)
 
 
+# TODO: This function is never used.
```
Comment (apssouza22) [inline server/src/load_data.py:28]: create_index and create_flat_index cand be deleted

File: server/src/load_data.py [hunk 2]
```diff
@@ -110,7 +111,7 @@ async def create_hnsw_index(
 
     application_field = TagField("application")
 
-    # Create index
+    # Create index (TODO: Compare the function `create_index()` above).
```
Comment (apssouza22) [inline server/src/load_data.py:114]: Remove this comment. This line has nothing to do with the create_index above

File: server/src/load_data.py (hunk 1 above)
Comment (vporton) [reply server/src/load_data.py]: Deleted. Note that I also deleted an invocation of `create_flat_index`. Hopefully, it's correct.

File: server/src/load_data.py (hunk 2 above)
Comment (vporton) [reply server/src/load_data.py]: I've removed the `TODO` part of the comment. Or to remove it entirely?

Comment (apssouza22) [reply server/src/load_data.py]: The To do

**Pull request merged**